### PR TITLE
Surrender while in Soft Crit

### PR DESCRIPTION
**Pull request by LovliestPlant**
## About The Pull Request
This PR allows the *surrender emote to be used while in soft crit, and adds a switch to provide an alternate message should the player already be on the floor.

## How This Contributes To The Skyrat Roleplay Experience
TG combat is rather fast, at times your first indication that you're in a losing fight is landing flat on your face. Expanding the options available to players in soft crit should result in fewer lethal ends to fights, allowing for more opportunities for RP at the end of combat.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

[Edited for brevity]

https://user-images.githubusercontent.com/107971606/205730722-287ea2ff-f97b-4c2a-8836-3166bf7f1d2f.mp4

</details>

## Changelog

:cl:
add: You can now *surrender while in soft crit.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/indicators/code/combat_indicator.dm [hunk 1]
```diff
@@ -198,3 +199,8 @@ GLOBAL_VAR_INIT(combat_indicator_overlay, GenerateCombatOverlay())
 		living_user.Paralyze(200)
 		living_user.remove_status_effect(/datum/status_effect/grouped/surrender, src)
 		living_user.set_combat_indicator(FALSE)
+
+/datum/emote/living/surrender/select_message_type(mob/user, intentional)
+	if(user.stat == SOFT_CRIT)
```
Comment (tf-4) [inline modular_skyrat/modules/indicators/code/combat_indicator.dm]:
```suggestion
	if(user.body_position == LYING_DOWN)
```

Comment (LovliestPlant) [reply modular_skyrat/modules/indicators/code/combat_indicator.dm]: Looks to be throwing an 'undefined var' error on that body_position. Not sure what the difference is though, as it did fine with user.stat?

Comment (LovliestPlant) [reply modular_skyrat/modules/indicators/code/combat_indicator.dm]: oh is it getting upset at 'mob/living/carbon/user' for body_position as opposed to 'mob/user' for stat?

Comment (tf-4) [reply modular_skyrat/modules/indicators/code/combat_indicator.dm]: yeah you're right, it's a var on `mob/living`. Just make a new check
```
var/mob/living/living_mob = user
if(living_mob?.body_position == LYING_DOWN)
```
Or whatever

**Review by tf-4 — APPROVED** [review]
Seems good to me

**Pull request merged**